lunabot_sim/lunabot_description/launch/controller.yaml

```diff
@@ -0,0 +1,23 @@
+dummy_bot_controller:
+  # Publish all joint states -----------------------------------
+  joint_state_controller:
+    type: joint_state_controller/JointStateController
+    publish_rate: 50
+
+  # Position Controllers --------------------------------------
+  right_front_rev_position_controller:
+    type: effort_controllers/JointPositionController
+    joint: right_front_rev
+    pid: { p: 100.0, i: 0.01, d: 10.0 }
```

Comment:
Do we want separete PID for each wheel? (Maybe we do, just a question)

Reply:
This is for gazebo. it requires a controller for each joint, unless its a group controller, but tbh not worth our time messing around with.